lms/templates/instructor/instructor_dashboard_2/send_email.html

```diff
@@ -5,108 +5,124 @@
 %>
```

Comment:
(To view without whitespace changes, click here)[https://github.com/edx/edx-platform/pull/12602/files?w=1#diff-0aafb7e04aabb73712746a2815573beb]